st2mistral/functions/stackstorm.py

```diff
@@ -18,17 +18,25 @@
 from oslo_config import cfg
 from oslo_log import log as logging
 
-from st2mistral import config
-config.register_opts()
-
 from mistral import exceptions as exc
 from st2mistral.utils import http
 
+from st2mistral import config
+config.register_opts()
```

Comment:
Why did you move this config section?  There's a reason for the config section to be up top.  It'll load config which will be used in the http module.